crank/models.py

```diff
@@ -1,11 +1,59 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
 
 # Create your models here.
 class Greeting(models.Model):
     when = models.DateTimeField('date created', auto_now_add=True)
 
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    email_confirmed = models.BooleanField(default=False)
+
+#begin Tony-Frank pair insert
+class Course_Listing_Table(models.Model):
+    courseid = models.IntegerField(primary_key=True)
+    coursenumber=models.CharField(max_length=10)
+    coursetitle=models.CharField(max_length=128)
+
+class Faculty_Table(models.Model):
+    facultyid = models.IntegerField(primary_key=True)
+    facultyname = models.CharField(max_length=128)
+
+class Course_Faculty_Table(models.Model):
+    course = models.ForeignKey(Course_Listing_Table)
+    faculty = models.ForeignKey(Faculty_Table)
+    course_faculty_pair = models.IntegerField(primary_key=True)
```

Comment:
Mind renaming this to have id in the name?

Also I’m guessing this is just an auto-incrementing number? Where is the code to do the incrementing?

Reply:
Same for the other id fields